src/odemis/gui/model/__init__.py

```diff
@@ -187,6 +187,9 @@ def __init__(self, microscope):
         # Indicates whether the microscope is acquiring a high quality image
         self.is_acquiring = model.BooleanVA(False)
 
+        # Indicates whether the microscope is performing an preparation process
```

Comment:
Reword to something in the lines of: Indicates whether a stream is in preparation (ie, a prepare() future is active)